addons/portal_rating/static/src/xml/portal_rating_composer.xml

```diff
@@ -39,7 +39,7 @@
                             </t>
                         </h5>
                         <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                            <span>&amp;times;</span>
+                            <span t-translation="off">×</span>
```

Comment:
in `static/xml` files, it seems it's even [8 characters minimum](https://github.com/odoo/odoo/blob/14.0/odoo/tools/translate.py#L848) which seems a very high limit :thinking: